get-pr-info/action.yml

```diff
@@ -52,16 +49,17 @@ runs:
   steps:
     - name: >
         Fetch information about pull request #${{ inputs.pull_request_number }} 
-        in ${{ inputs.owner}}/${{ inputs.repo }}
+        in ${{ inputs.repository }}
       id: get-info
       uses: actions/github-script@v6
       continue-on-error: false
       with:
         github-token: ${{ inputs.token }}
         script: |
+          const fullRepoName = "${{ inputs.repository }}".split("/")
```

Comment:
same as in the other file

Reply:
Fixed.

Reply:
Reverted this back since the suggested variant doesn't work. The URL becomes `https://api.github.com/repos/tarantool%2Ftarantool/pulls/7573` which is wrong (%2 in the path).